packages/apollo-voyager-keycloak/src/AuthContextProvider.ts

```diff
@@ -0,0 +1,31 @@
+import { Request } from "express"
+import { IncomingMessage } from "http"
+
+export default class KeycloakAuthContextProvider {
+
+  public readonly request : any
+  public readonly accessToken : any
+  public readonly authenticated: boolean
+
+  constructor (request: any) {
+    this.request = request
+    this.accessToken = (request && request.kauth && request.kauth.grant) ? request.kauth.grant.access_token : undefined
```

Comment:
Looks like keycloak is not exposing this (seems to be internal api): https://github.com/keycloak/keycloak-js-bower/blob/master/dist/keycloak.d.ts

We should stick with any